Sell/Payment/Repository/PaymentProvidable.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  PaymentProvidable.swift
+//  RealifeTech
+//
+//  Created by Mickey Lee on 30/06/2021.
+//  Copyright © 2021 Realife Tech. All rights reserved.
+//
+
+import Foundation
+
+public protocol PaymentProvidable {
```

Comment:
May I know will we define `deleteMyPaymentSource` at this stage please?

Reply:
Yes, we will and we should. I forgot to add it to the architecture document. Thanks for your reminder.